scripts/egghead-course-notes.js

```diff
@@ -9,17 +9,25 @@ let { queryEggheadCourse } = await lib("egghead");
 
 let octokit = new Octokit();
 export let getCourseNotesContentByPath = async (path) => {
-  let { data } = await octokit.rest.repos.getContent({
-    owner: "eggheadio",
-    repo: "eggheadio-course-notes",
-    path,
-  });
-
-  return data;
+  try{
+    let { data } = await octokit.rest.repos.getContent({
+      owner: "eggheadio",
+      repo: "eggheadio-course-notes",
+      path,
+    });
+    return data;
+  } catch (err) {
+    if(err.status === 404){
+      console.log(`Notes folder '${path}' does not exist. Check the file structure of the Course Notes you are trying to publish.`)
+    }
+  }
```

Comment:
Error handling. If the `path` doesn't exist in the course notes repo, it will return a 404. Users can try again or exit out of the script.